packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/TransferDetails/template.success.tsx

```diff
@@ -170,53 +195,74 @@ const Success: React.FC<Props> = props => {
             </Copy>
           </RowCopy>
         )}
+        {(props.fiatCurrency === 'USD' || props.fiatCurrency === 'GBP') &&
+          props.account.agent.account !== '' && (
```

Comment:
Change to !props.account.agent.account

Reply:
We have `props.account.agent.account` and it's empty string, that is why checking if it's diff then empty string, otherwise we will have 
![image](https://user-images.githubusercontent.com/67264242/92590689-ee984d00-f29c-11ea-8ad0-fad5aa86f83f.png)

Reply:
<img width="72" alt="Screen Shot 2020-09-09 at 2 34 20 PM" src="https://user-images.githubusercontent.com/5640772/92598771-8dc34180-f2a9-11ea-8cc8-d54c52878bf9.png">

Reply:
@plondon ok you mean then on `!!props.account.agent.account` not on `!props.account.agent.account`